trueusd/README.md

```diff
@@ -0,0 +1,20 @@
+# Chainlink External Adapter for trueusd
```

Comment:
Let's get this updated, no input params, but then take an example for the output body.